ports/rp2/mpbthciport.h

```diff
@@ -30,13 +30,11 @@
 void mp_bluetooth_hci_init(void);
 
 // Poll the HCI now, or after a certain timeout.
-void mp_bluetooth_hci_poll_now(void);
+void mp_bluetooth_hci_schedule_poll(void);
 void mp_bluetooth_hci_poll_in_ms(uint32_t ms);
 
 // Must be provided by the stack bindings (e.g. mpnimbleport.c or mpbtstackport.c).
-// Request new data from the uart and pass to the stack, and run pending events/callouts.
-// This is a low-level function and should not be called directly, use
-// mp_bluetooth_hci_poll_now/mp_bluetooth_hci_poll_in_ms instead.
+// Request new data and pass it to the stack, and run pending events/callouts.
```

Comment:
Maybe "Request new HCI data ...".

I also think the other comment lines here should be retained.